Addition of clear() causes libstdc++ not to bootstrap on i386.
Until this is resolved, I'm removing it again.

--- libstdc++/std/bastring.h.jj	Sat Jul 15 19:53:09 2000
+++ libstdc++/std/bastring.h	Mon Mar 13 12:02:40 2000
@@ -346,8 +346,6 @@ public:
     { resize (n, eos ()); }
   void reserve (size_type) { }
 
-  void clear() { erase(begin(), end()); }
-
   size_type copy (charT* s, size_type n, size_type pos = 0) const;
 
   size_type find (const basic_string& str, size_type pos = 0) const
